airflow-core/src/airflow/serialization/serde.py

```diff
@@ -365,30 +365,29 @@ def _register():
     _stringifiers.clear()
 
     with Stats.timer("serde.load_serializers") as timer:
-        for _, name, _ in iter_namespace(airflow.serialization.serializers):
-            name = import_module(name)
-            for s in getattr(name, "serializers", ()):
-                if not isinstance(s, str):
-                    s = qualname(s)
-                if s in _serializers and _serializers[s] != name:
-                    raise AttributeError(f"duplicate {s} for serialization in {name} and {_serializers[s]}")
+        for _, module_name, _ in iter_namespace(airflow.serialization.serializers):
+            module = import_module(module_name)
+            for serializers in getattr(module, "serializers", ()):
+                s = serializers if isinstance(serializers, str) else qualname(serializers)
```

Comment:
```suggestion
                serializers_qualname = serializers if isinstance(serializers, str) else qualname(serializers)
```

nit: `s` or `d` don't fit my taste. or `s_qualname` might be better 🤔

Reply:
Good point, applied!